00_Core/Scripts/Source/MilkQUEST.psc

```diff
@@ -594,9 +599,18 @@ Function MilkCycle(Actor akActor, int t)
 
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.LactacidCount", LactacidCnt)
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.MilkGen", MaidMilkGen)
-	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.MilkCount", MilkCnt)
+	MME_Storage.setMilkCurrent(akActor, MilkCnt)
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.PainCount", PainCnt)
 
+	; setMilkCurrent() automatically restricts the provided value to the maximum allowed value
```

Comment:
Sorry, I totally forgot about this. I can add this to the patch set and update the pull request.

On a more general note: Do you like the idea of having actor-specific/customizable limits?